src/extension/notebook.ts

```diff
@@ -1,22 +1,20 @@
 import {
-  NotebookSerializer, ExtensionContext, Uri, workspace, NotebookData, NotebookCellData, NotebookCellKind
-} from 'vscode'
+  NotebookSerializer, ExtensionContext, Uri, workspace, NotebookData, NotebookCellData, NotebookCellKind,
+  window} from 'vscode'
```

Comment:
```suggestion
  NotebookSerializer, ExtensionContext, Uri, workspace, NotebookData, NotebookCellData, NotebookCellKind, window} from 'vscode'
```

Reply:
not ready for review. we'll fix all this stuff when we get close to finalizing.